.travis.yml

```diff
@@ -121,6 +121,7 @@ jobs:
         HOST=x86_64-unknown-linux-gnu
         PACKAGES="python3-zmq qtbase5-dev qttools5-dev-tools protobuf-compiler libdbus-1-dev libharfbuzz-dev libprotobuf-dev"
         DEP_OPTS="NO_QT=1 NO_UPNP=1 DEBUG=1 ALLOW_HOST_PACKAGES=1"
+        TEST_RUNNER_EXTRA="--coverage --extended --exclude feature_dbcrash"
```

Comment:
What's purpose of excluding feature_dbcrash? Can you add a comment in the code saying why it's needed?

It would also be nice to know about why the extended flag is added, if adding it is intentional.

Reply:
Done